eslintrc.json

```diff
@@ -29,7 +29,7 @@
     "key-spacing": ["error", {"beforeColon": false, "afterColon": true}],
     "keyword-spacing": "error",
     "max-depth": "off",
-    "max-len": ["warn", 90, 8, {"ignoreUrls": true}],
+    "max-len": ["warn", 90, 8, {"ignoreUrls": true, "ignorePattern": "^\\s*(QUnit\\.)?test\\(.*"}],
```

Comment:
oh, you can ignore specific lines like this?
![neat](http://i.imgur.com/pEUfrLv.gif)